tools/clickhouse

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -euo pipefail
+
+usage='
+tools/clickhouse: start a clickhouse local server and print the app flags needed to connect to it
+
+Example:
+  $ bazel run enterprise/server -- $(tools/clickhouse)
+'
+
+# Env vars:
+
+# Optionally set to "podman" to use podman instead
+: "${BB_CLICKHOUSE_CONTAINER_RUNNER:=docker}"
+# Port that clickhouse server will listen on
+: "${BB_CLICKHOUSE_PORT:=9000}"
+# Clickhouse image to use
+: "${BB_CLICKHOUSE_IMAGE:=docker.io/yandex/clickhouse-server:latest}"
```

Comment:
Can we use version 22.3? the same as the production.

Reply:
done